core/src/main/scripts/importer/validateData.py

```diff
@@ -2872,10 +2872,24 @@ class TimelineValidator(Validator):
         'EVENT_TYPE']
     REQUIRE_COLUMN_ORDER = True
     ALLOW_BLANKS = True
+    NULL_VALUES = ['', "NA"]
 
     def checkLine(self, data):
         super(TimelineValidator, self).checkLine(data)
         # TODO check the values
+        for col_index, col_name in enumerate(self.cols):
+            # treat cells beyond the end of the line as blanks,
+            # super().checkLine() has already logged an error
+            value = ''
+            if col_index < len(data):
+                value = data[col_index].strip()
+            if col_name == 'START_DATE':
+                if value.strip() in self.NULL_VALUES:
```

Comment:
following up on @sheridancbio 's comment: START_DATE in DB (see https://github.com/cBioPortal/cbioportal/blob/master/db-scripts/src/main/resources/cgds.sql#L775) is `int NOT NULL`. So in case of `na`, the code would crash during importing. Perhaps better that testing `if value.strip() in self.NULL_VALUES` is to test whether it is a valid integer, and give the error if not.

Reply:
@pieterlukasse sorry for the late reply, I've updated it to check that the value consists of digits